NovetusLauncher/NovetusFuncs/CryptoRandom.cs

```diff
@@ -19,6 +20,19 @@ public CryptoRandom()
 		r = RandomNumberGenerator.Create();
 	}
 	
+	private static string DiogenesCrypt(string word)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'DiogenesCrypt'.](https://app.codacy.com/manual/Bitl/Novetus_src/pullRequest?prid=5476561)

Reply:
This is added so that a frontend form can be added later. Removing this voids the entire purpose of the pull request.